src/gluonnlp/data/word_embedding_evaluation.py

```diff
@@ -589,7 +627,7 @@ class BakerVerb143(WordSimilarityEvaluationDataset):
     _verify_ssl = False  # ie.technion.ac.il serves an invalid cert as of 2018-04-16
 
     min = 0
-    max = 10
+    max = 1
```

Comment:
@leezu would this affect the evaluation results?

Reply:
@szha, I'm not aware of any code relying on this attribute. So at least for the code in GluonNLP, correcting this value will not make any problems. Thanks for spotting it!